src/hidimstat/_utils/utils.py

```diff
@@ -105,3 +140,54 @@ def seed_estimator(estimator, random_state=None):
                 setattr(value, "random_state", RandomState(rng.bit_generator))
 
     return estimator
+
+
+def check_statistical_test(statistical_test):
+    """
+    Validates and returns a test statistic function.
+
+    Parameters
+    ----------
+    test : str or callable
+        If str, must be either 'ttest' or 'wilcoxon'.
+        If callable, must be a function that can be used as a test statistic.
+
+    Returns
+    -------
+    callable
+        A function that can be used as a test statistic.
+        For string inputs, returns a partial function of either ttest_1samp or wilcoxon.
+        For callable inputs, returns the input function.
+
+    Raises
+    ------
+    ValueError
+        If test is a string but not one of the supported test names ('ttest' or 'wilcoxon').
+    ValueError
+        If test is neither a string nor a callable.
+    """
+    if isinstance(statistical_test, str):
+        if statistical_test == "ttest":
+            return partial(ttest_1samp, popmean=0, alternative="greater", axis=1)
+        elif statistical_test == "wilcoxon":
+            return partial(wilcoxon, alternative="greater", axis=1)
+        elif statistical_test == "nb-ttest":
+            return partial(
+                nadeau_bengio_ttest,
+                popmean=0,
+                test_frac=0.1 / 0.9,
```

Comment:
This shouldn't be hardcoded. It should depend on the data passed to fit and importance.

Reply:
However, nadeau_bengio_ttest doesn't provide theoretically guarantees.